src/Entity/PhoneNumberSettings.php

```diff
@@ -85,7 +85,7 @@ class PhoneNumberSettings extends ConfigEntityBase implements PhoneNumberSetting
    *
    * @var bool
    */
-  protected $verification_phone_number_purge = TRUE;
+  protected $purgeVerificationPhoneNumber = TRUE;
```

Comment:
it makes sense English wise, but the property was named this way because of the 'verification' prefix, indicating a category of properties.